models/Recipe.model.js

```diff
@@ -2,7 +2,40 @@ const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
-  // TODO: write the schema
+  title: {
+    type: String,
+    required: true,
+  },
+  level: {
+    type: String,
+    enum: ['Easy Peasy', ' Amateur Chef', 'UltraPro Chef']
+  },
+  ingredients: {
+    type: [String],
+  },
+  cuisine: {
+    type: String,
+    required: true,
+  },
+  dishType: {
+    type: String,
+    enum: ['breakfast', 'main_course', 'soup', 'snack', 'drink', 'dessert', 'other']
+  },
+  image: {
+    type: String,
+    default: "https://images.media-allrecipes.com/images/75131.jpg",
+  },
+  duration: {
+    type: Number,
+    min: 0,
+  },
+  creator:{
+   type: String,
+  }, 
+  created:{ 
+    type: Date,
+    default: 23/06/2021, 
```

Comment:
Hoy esto ya no vale. Deberías haber visto venir este problema.

```suggestion
    default: Date.now, 
```